data/wanderers middle.txt

```diff
@@ -1768,6 +1768,85 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+			choice
+				`	"How did your ship end up in such a state?"`
+				`	"Was this beauty the victim of a pirate attack?"`
+				
+			`	"I'm not quite sure," the captain replies. "We were shot by a fleet of hostile ships that our computer couldn't identify. They didn't look at all like a run-of-the-mill pirate ship, though, and they were kitted out with tech I've never seen before."`
+			choice
+				`	"How many ships were there? What did they look like?"`
+				`	"Where did this fight take place?"`
+					goto where
+				`	"You should be glad you're still in one piece."`
+					goto glad1
+					
+			`	"Well," the pilot continues, "we had hired a beefy Leviathan escort to Farpoint in the Alnitak system. When we were going in for landing, two of these hostile ships jumped in from the northwest, the direction of all the anarchist colonies in this area. The ships weren't big, but they were tough: our four heavy laser turrets weren't enough to produce even a dent in their shields. They made short work of the Leviathan, and nearly disabled our ship, too. It was lucky I decided that jumping would be faster than landing. We warped out of the system while in atmosphere, saving the ship, but now I'm in debt to all my crew because I couldn't meet a rush delivery deadline."`
+			choice
+				`	"You're lucky as hell to be here today."`
+					goto glad
+				`	"Thanks for the information. I think that these hostile ships are the ones I'm looking for."`
+					goto thanks
+					
+			label where
+			`	"Well," the pilot explains, "we were doing a large cargo run to Farpoint in the Alnitak system, when two of these hostile ships jumped in from the northwest, the direction of all the anarchist colonies in this area. I had to jump the ship in atmosphere to stop us from getting shredded by their strange melting weaponry. I immediately reported the attack to a desk clerk at the Navy base in the neighboring system Saiph, but their servers show no logs of any unidentified ships beyond Farpoint, so I guess they turned around after scouting out the Alnitak system."`
+			choice
+				`	"You're lucky as hell to be here today."`
+					goto glad
+				`	"Thanks for the information. I think that these hostile ships are the ones I'm hunting."`
+					goto thanks
+			
+			label glad1
+			`	"I am! The attacking ships weren't big, but they were tough: our four heavy laser turrets couldn't even produce a dent in their shields," the captain explains. "We even hired a beefy Leviathan to protect us from the pirate raids as we travelled to Farpoint, but when these hostile ships jumped in from the northwest, they made short work of our escort and nearly disabled our ship, too. I had to jump out of the system while in atmosphere to save the ship, but now I'm in debt to all my crew because I didn't meet a rush delivery deadline."`
```

Comment:
Most writing guide recommend just using "say" and "ask" as [speech tags](http://www.scribophile.com/academy/he-said-she-said-dialog-tags-and-using-them-effectively) rather than using a variety of words like, "explains," "continues," "replies," etc. Otherwise you end up drawing attention to the tags instead of to the dialog itself.

When writing text for the game and especially when writing dialog, you should also ask yourself whether each sentence is interesting enough to be worth the player's time. For example, the last sentence here could be left out.